test/unit/template_importer_test.rb

```diff
@@ -92,5 +92,15 @@ def importer(opts = {})
       assert_equal File.read(get_template('ptable1.erb')), pt.layout
       assert_equal 'Debian', pt.os_family
     end
+
+    test 'purge! removes expected template' do
+      FactoryGirl.create(:provisioning_template, :name => 'Community delete_me')
+      FactoryGirl.create(:provisioning_template, :name => 'keep_me')
+      @importer.purge!
+      ct = ProvisioningTemplate.find_by_name('delete_me')
+      assert not ct.present?
```

Comment:
Cool, I was looking at the a assertion stuff for a negative test, obviously missed refute, thanks for that.